lcm/src/message.rs

```diff
@@ -109,14 +109,14 @@ impl Marshall for String {
 
         let len = i32::decode(buffer)?;
         if len <= 0 {
-            return Err(DecodeError::invalid_size(len as i64));
+            return Err(DecodeError::invalid_size(i64::from(len)));
```

Comment:
This one is also an interesting change.

Reply:
This is suggested because `as` could truncate if `len` were to become larger or unsigned in future.  Using from makes the infallibility of the conversion obvious to the reader.

Reply:
That's correct.